### ISSUE TITLE
The REPL console has no syntax highlighting

### ISSUE THREAD

**Issue by ericdallo** [afucher/clojure-repl-intellij#18]

**Comment by gerdint** [comment]
Paredit bindings would also be nice.

**Comment by ericdallo** [comment]
@gerdint paredit bindings are already available in clojure-lsp-intellij, check it [here](https://github.com/clojure-lsp/clojure-lsp-intellij/blob/master/docs/features.md#paredit)

**Comment by gerdint** [comment]
@ericdallo yeah I know, but they do not seem to work in the REPL, as it stands.

**Comment by afucher** [comment]
To support highlight we will probably work with a Clojure buffer, so I think that all things available in a common clojure file will be available there.

**Comment by nicollecnunes** [comment]
even with `clojure-lsp-intellij` my repl does not have syntax highlighting nor paredit bindings, am i missing some config?

**Comment by gerdint** [comment]
@nicollecnunes No. This issue is still to open. Like @afucher says some major replumbing probably needs to be done.

@afucher Personally I prefer a split REPL-buffer with separate windows panes for REPL output and input.
